### PR TITLE
SoW - High House of Light

### LINKED ISSUE #3811: SoW - High House of Light
image

### PR DESCRIPTION
**Pull request by marcsoiferman-cm**
Closes #3811

### CODE REVIEW

File: server/game/cards/12-SoW/HighHouseOfLight.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+                        cannot: 'target',
+                        restricts: 'opponentsEvents'
+                    })
+                }))
+            },
+            then: context => ({
+                condition: context => this.game.currentConflict.getNumberOfCardsPlayed(context.player) >= 5,
+                gameAction: AbilityDsl.actions.selectRing({
+                    activePromptTitle: 'Choose a ring to take a fate from',
+                    ringCondition: ring => ring.fate >= 1,
```
Comment (jeremylarner) [inline server/game/cards/12-SoW/HighHouseOfLight.js]: This shouldn't be necessary

File: server/game/cards/12-SoW/HighHouseOfLight.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+                    message: '{0} moves a fate from the {1} to {2}',
+                    messageArgs: ring => [context.player, ring, context.target],
+                    gameAction: AbilityDsl.actions.multiple([
```
Comment (jeremylarner) [inline server/game/cards/12-SoW/HighHouseOfLight.js]: I think you want a single gameAction here: `AbilityDsl.actions.placeFate({ target: context.target })`, and you need to add `subActionProperties: ring => { origin: ring }`

File: server/game/cards/12-SoW/HighHouseOfLight.js (hunk 1 above)
Comment (marcsoiferman-cm) [reply server/game/cards/12-SoW/HighHouseOfLight.js]: tested, it's required.  Otherwise can select a ring with no fate

Comment (jeremylarner) [reply server/game/cards/12-SoW/HighHouseOfLight.js]: Pretty sure that's a bug in SelectRingAction - issue raised here #3850

**Pull request merged**